contents/tutorials/non-technical-guide-to-data.md

```diff
@@ -0,0 +1,77 @@
+---
+title: A non-technical guide to understanding data in PostHog
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-11-28
+featuredImage: ../images/tutorials/banners/non-technical-guide-to-data.png
+topics: ["data management"]
+---
+
+Non-technical users have many tools in PostHog for gaining insights from their product data. You don’t need coding knowledge, but you do need some knowledge of your data. Knowing the formatting and structure of your data is key to getting the most out of PostHog as a non-technical user.
+
+In this tutorial, we’ll go through how you can understand your product’s data in PostHog. This allows you to find the data you want, format it in the right way, and generate insights to improve your product. 
+
+> This tutorial assumes your technical team set up PostHog in your app or site, and you have data to work with. Read our [event tracking guide](/tutorials/event-tracking-guide) for details on setting this up if not.
+
+## 1. Explore other’s creations
+
+In school, using someone else’s work is cheating, but in real life, it is “collaboration.” Exploring teammates’ work in PostHog is a great way to get started understanding the data there.
```

Comment:
```suggestion
In school, using someone else’s work is cheating; in real life, it's “collaboration.” Exploring teammates’ work in PostHog is a great way to get started understanding the data there.
```

This made me lol. I'd suggest engaging the awesome power of the semi-colon to make it hit even better. Semi-colons are gangster.